docs/modules/ROOT/pages/api/utilities.adoc

```diff
@@ -2,8 +2,7 @@
 
 :deploy_syscall: link:https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/system-calls-cairo1/#deploy[deploy_syscall]
 
-The following documentation provides reasoning and examples for functions and constants found in `openzeppelin::utils`
-and `use openzeppelin::utils::test_utils;`.
+The following documentation provides reasoning and examples for functions and constants found in `openzeppelin_utils`.
```

Comment:
We can keep this (and the rest of the doc) as it was with `openzeppelin::utils`. The whole library will still be available in the `openzeppelin` package

Reply:
I've changed it back to `openzeppelin::utils`, otherwise we would have to change the module names everywhere for consistency. The good thing about `openzeppelin_utils` is that it will always work, either with `openzeppelin` dependency or with `openzeppelin_utils` as a standalone package

Reply:
> The good thing about openzeppelin_utils is that it will always work

Did you test this? If we import only openzeppelin, will `openzeppelin_utils` work instead of `openzeppelin::utils`?

Reply:
And did you change it back to `openzeppelin::utils`? I'm seeing `openzeppelin_utils` in the PR instead.

Reply:
1. Yes, I've checked it and it works both ways if `openzeppelin` is used as a dependency
2. Indeed, please check again, I've just pushed the changes